### ISSUE TITLE
More description needed in blueprints

### ISSUE THREAD

**Issue by ksnyde** [html-next/flexi#40]
I don't mind succinct documentation but I can't find my way out of the box here:

> ![image](https://cloud.githubusercontent.com/assets/980757/14183490/827d908c-f768-11e5-9f00-6d67968c2f57.png)

what does `/path/to/<name>` refer to? I assumed it meant the route, component, etc. but when trying this it complained saying that the only valid names were the breakpoints. That's forcing me to think about things differently than I expected. 

Are the templates only for the full page? I would have thought you could have templates on a per component basis.

**Comment by runspired** [comment]
Templates do work for components or routes, but the generators that are specific to `component` or `route` are unfinished (see #12 ).

This `ember g layout` works with either routes or components, the `path/to` part should be the path to the component or route including the name of the component or route, while `/name should be the name of a breakpoint.  You can also convert existing templates to a layout using `move:layout` or `mvl` if operating off of develop or the about to be released patch, I'll be adding documentation on this shortly.

```cli
ember move:layout <type> <name> [-b <breakpoint>] [-v]
```

**example**
```cli
ember mvl route foo/bar -b "desktop"
```

 ```cli
ember mvl component foo/bar -b "mobile"
```
 
If you define `defaultBreakpoint` in `config/flexi.js` it will be used for `-b` if none is provided

`-v` is verbose mode, will show you the paths being resolved

If you are moving a route template instead of component, or a pod component that's not within the components/ directory, you can do this (edited)

```cli
ember mvl foo/bar -b "desktop"
```

`mvl` is a shorthand alias for `move:layout`

**Comment by ksnyde** [comment]
Thanks Chris.

**Comment by rwwagner90** [comment]
I have copied this documentation into the readme.io docs, which will be released as part of the 2.0 release, so I am closing this.
